CHANGELOG.md

```diff
@@ -8,6 +8,102 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Managed Simple Data Exchanger (SDE)](https://github.com/eclipse-tractusx/managed-simple-data-exchanger) | sde: [0.1.11](https://github.com/eclipse-tractusx/managed-simple-data-exchanger/releases/tag/sde-0.1.11) | frontend: [2.4.2](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/releases/tag/v2.4.2)<br>backend: [2.4.2](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/releases/tag/v2.4.2) |
+| [PCF Exchange Kit](https://eclipse-tractusx.github.io/docs-kits/category/pcf-exchange-kit) | n/a | [1.2.0](hthttps://eclipse-tractusx.github.io/docs-kits/kits/PCF%20Exchange%20Kit/PCF%20Exchange%20Kit%20Changelog) |
+| [Policy Hub](https://github.com/eclipse-tractusx/policy-hub) | policy-hub: [1.1.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.1.0) | [1.1.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/v1.1.0) |
+| [Portal](https://github.com/eclipse-tractusx/portal) | portal: [2.1.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.1.0) | registration: [2.0.1](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.1)<br>frontend: [2.1.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.1.0)<br>backend: [2.1.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.1.0)<br>assets: [2.1.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.1.0) |
+| [PURIS](https://github.com/eclipse-tractusx/puris) | puris: [2.7.0](https://github.com/eclipse-tractusx/puris/releases/tag/puris-2.7.0) | [2.1.0](https://github.com/eclipse-tractusx/puris/releases/tag/2.1.0) |
+| [PURIS Kit](https://eclipse-tractusx.github.io/docs-kits/category/puris-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/kits/PURIS%20Kit/PURIS%20Kit%20Changelog) |
+| [Self Description (SD) Factory](https://github.com/eclipse-tractusx/sd-factory) | sdfactory: [2.1.21](https://github.com/eclipse-tractusx/sd-factory/releases/tag/sdfactory-2.1.21) | [2.1.13](https://github.com/eclipse-tractusx/sd-factory/releases/tag/v2.1.13) |
+| [Semantic Hub](https://github.com/eclipse-tractusx/sldt-semantic-hub) | semantic-hub: [0.3.1](https://github.com/eclipse-tractusx/sldt-semantic-hub/releases/tag/semantic-hub-0.3.1) | [0.4.0](https://github.com/eclipse-tractusx/sldt-semantic-hub/releases/tag/v0.4.0) |
+| [SSI Credential Issuer](https://github.com/eclipse-tractusx/ssi-credential-issuer) | ssi-credential-issuer: [1.1.0](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/ssi-credential-issuer-1.1.0) | [1.1.0](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/v1.1.0) |
+| [Trace-X Traceability Application](https://github.com/eclipse-tractusx/traceability-foss) | helm-charts: [1.3.43](https://github.com/eclipse-tractusx/traceability-foss/releases/tag/helm-charts-1.3.43) | [13.0.1](https://github.com/eclipse-tractusx/traceability-foss/releases/tag/13.0.1) |
```

Comment:
But in some, there is no "v" in the version. To be consistent I would add also the version with "v" with it is there and without when it not? 

@stephanbcbauer and @tom-rm-meyer-ISST what do you think?

Reply:
In previous versions it was done like that: 
![image](https://github.com/user-attachments/assets/d3166320-dc31-42b0-a0e0-cdc30050f3bd)

Reply:
The `v` is inconsistent because we didn't check that as hard criteria in QGates + I think I saw it differently in TRGs (docker publishing workflow or something like that). 

We now only used the v in a version if 
a) the respective product uses it
b) it's an app version

Reply:
it was an app version and the products are using it, because of this I updated it

Reply:
for me that's fair. Seems like I was thinking out aloud - sorry for that :D 

Solved from my point of view

Reply:
I don't like the v in front :), but it is what it is :)